### PR TITLE
Add fsc.export

### PR DESCRIPTION
**Pull request by chrisjsewell**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chrisjsewell** [comment]
@conda-forge/help-python this is ready for review thanks

File: recipes/fsc.export/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+test:
+  imports:
+    - fsc.export
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://frescolinogroup.github.io/frescolino/pyexport/1.2
```
Comment (saraedum) [inline recipes/fsc.export/meta.yaml:33]:
```suggestion
  home: http://frescolinogroup.github.io/frescolino/pyexport/1.2
  dev_url: https://github.com/frescolinogroup/pyexport
```

File: recipes/fsc.export/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  summary: Decorator to add objects to their module's __all__.
+  license: Apache-1.1
```
Comment (saraedum) [inline recipes/fsc.export/meta.yaml:35]: Could you clarify where it says that this is Apache 1.1 licensed? I only see a generic "apache license" in the source code.

**Review by saraedum — CHANGES_REQUESTED** [review]

File: recipes/fsc.export/meta.yaml (hunk 2 above)
Comment (chrisjsewell) [reply recipes/fsc.export/meta.yaml:36]: Ah this was just auto-added by https://github.com/conda-incubator/grayskull, can change it to just `Apache` if you want?

File: recipes/fsc.export/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://frescolinogroup.github.io/frescolino/pyexport/1.2
+  dev_url: https://github.com/frescolinogroup/pyexport
+  summary: Decorator to add objects to their module's __all__.
+  license: Apache-1.1
```
Comment (chrisjsewell) [inline recipes/fsc.export/meta.yaml:36]:
```suggestion
  license: Apache
```

File: recipes/fsc.export/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://frescolinogroup.github.io/frescolino/pyexport/1.2
+  dev_url: https://github.com/frescolinogroup/pyexport
+  summary: Decorator to add objects to their module's __all__.
+  license: Apache
```
Comment (chrisjsewell) [inline recipes/fsc.export/meta.yaml:36]:
```suggestion
  license: Apache-2.0
```

File: recipes/fsc.export/meta.yaml (hunk 2 above)
Comment (chrisjsewell) [reply recipes/fsc.export/meta.yaml]: changes to apache-2.0

Comment (saraedum) [reply recipes/fsc.export/meta.yaml]: Sorry, but do we know whether this is apache 2.0?

Comment (chrisjsewell) [reply recipes/fsc.export/meta.yaml]: Well I guess we don't exactly; but what does a non-versioned Apache license default to? Or are you saying that it cannot be accepted until the license is changed at source?

Comment (saraedum) [reply recipes/fsc.export/meta.yaml]: It's my understanding that these licenses are quite different. Can we check with upstream what the license actually is?

If upstream does not respond, we can discuss what to make of this but then we probably need to include more people (I am not sure what's the conda-forge policy here.)

Comment (chrisjsewell) [reply recipes/fsc.export/meta.yaml]: Hey @saraedum, I checked what the license is here: https://github.com/greschd/aiida-tools/issues/3; it was apache 2.0, and I have updated it as suggested there

Comment (chrisjsewell) [reply recipes/fsc.export/meta.yaml]: is that ok?

**Pull request closed without merging**